Sources/PointFree/Episode/Show.swift

```diff
@@ -339,10 +339,25 @@ let dividerView = View<Prelude.Unit> { _ in
 
 private let transcriptView = View<[Episode.TranscriptBlock]> { blocks in
   div([`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]]), Class.pf.colors.bg.white])],
-      blocks.flatMap(transcriptBlockView.view)
+      blocks.filter((!) <<< ^\.type.isExercise).flatMap(transcriptBlockView.view)
```

Comment:
I'm filtering them from the inline view for now since I'm not sure how to style them nicely.